client/src/store.js

```diff
@@ -94,18 +96,17 @@ export async function questionData(qid, qs) {
 
 	// give the next batch a way to wait for us to complete
 	let resolve;
-	let reject;
-	fetch_done = new Promise((resolve1, reject1) => {
+	fetch_done = new Promise((resolve1) => {
 		resolve = resolve1;
-		reject = reject1;
```

Comment:
I think we're going to start using `reject` here in the future, so I'd rather leave the machinery in place for now. Is there a way to have the linter not fail on this particular unused thing? Maybe by prefixing it with `_` or something?

Reply:
Sounds good. I'll see if our eslint config allows the `_` prefix trick (and I can enable it if it doesn't since it will feel familiar to `Rustaceans`). But either way I can also just disable the `unused-variable` error for those specific lines to allow the linter to pass for now.

Reply:
Enabled `_` variable in [c5264de](https://github.com/jonhoo/wewerewondering/pull/11/commits/c5264de2064d5ac398a24934699ea17a93cdc027) and reintroduced `reject` as  `_reject` in [2966a1e](https://github.com/jonhoo/wewerewondering/pull/11/commits/2966a1e7f7cb33157b72a7d0cb4af813d1eb6cc2)